Homework, lecture 4.py

```diff
@@ -0,0 +1,135 @@
+#Homework 4.1 have problems with this tasks.. 
+
+#Homework 4.2
+
+def is_year_leap(year):
+    if year == 2000 or year == 2016: 
+        return True
+    else: 
+        return False 
+
```

Comment:
NO. Here should be placed algorithm like this:
```suggestion
def is_leap_year(year): 
    if year % 4 != 0:
        return False
    elif year % 100 != 0:
        return True
    elif year % 400 != 0:
        return False
    else:
        return True
```